src/models/converters/transactions/mod.rs

```diff
@@ -211,6 +212,34 @@ impl MultisigTransaction {
             .flatten()
             .unwrap_or(0)
     }
+
+    fn is_cancellation(&self) -> bool {
+        self.to == self.safe
+            && data_size(&self.data) == 0
+            && self.value.as_ref().map_or(true, |value| value == "0")
+            && self
+                .operation
+                .map_or(true, |operation| operation == Operation::CALL)
+            && self
+                .base_gas
+                .as_ref()
+                .map_or(true, |base_gas| base_gas.eq(&0))
+            && self
+                .gas_price
+                .as_ref()
+                .map_or(true, |gas_price| gas_price == "0")
+            && self.gas_token.as_ref().map_or(true, |gas_token| {
+                gas_token.len() < 2 || gas_token == "0x0000000000000000000000000000000000000000"
```

Comment:
why `< 2`? Actually I would say the equals is enough. I know web regards `0x` as a valid address ... but it is actually not :P

Reply:
ah yeah, that's what I was afraid. Cool I will only the equals then :)